src/parser/data/Types.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Robin Textor-Falconi (textorr@informatik.uni-freiburg.de)
+
+#pragma once
+
+#include <array>
+#include <vector>
+
+#include "./VarOrTerm.h"
+
+namespace Types {

```

Comment:
1. namespaces are `lower_case`
2. Is this rather namespace ad_utility::sparql_types ?

( They are mostly connected to the sparql grammar and this should be reflected in the namespace).